Ipr review cleanup

rabardini opened the pull request.

stephanesan commented:
Should not have done this on my master. Will move this to a branch and make a new pull request.

marktrayer committed:
Merge pull request #50 from rabardini/master

Updates to align language support to the specification text

marktrayer committed:
Merge pull request #52 from rabardini/master

Fixes for the /oic/res resource to conform to the specification text

marktrayer committed:
Cleanup of RAML and JSON to address errors identified by raml2doc

marktrayer committed:
Including changes from Richard.
Merge branch 'IPR-Review-Cleanup' of https://github.com/openconnectivityfoundation/core into IPR-Review-Cleanup

marktrayer committed:
Correct the id in the schema for oic.r.icon

marktrayer committed:
Adding introspection JSON and RAML (after cleanup)

marktrayer committed:
Adding JSON and RAML for introspection
